Profiles/cio-Symptom.xml

```diff
@@ -0,0 +1,153 @@
+      </mapping>
+    </element>
+    <element id="Observation.extension">
+      <path value="Observation.extension" />
+      <min value="1" />
+    </element>
+    <element id="Observation.extension:medicationHypersensitivityIdentifier">
+      <path value="Observation.extension" />
+      <sliceName value="medicationHypersensitivityIdentifier" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivityIdentifier" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.extension:medicationHypersensitivityIdentifier.url">
+      <path value="Observation.extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-MedicationHypersensitivityIdentifier" />
+    </element>
+    <element id="Observation.extension:relationCondition">
+      <path value="Observation.extension" />
+      <sliceName value="relationCondition" />
+      <min value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-RelationCondition" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Observation.identifier">
+      <path value="Observation.identifier" />
+      <short value="IdentificationNumber" />
+      <definition value="Globally unique number that identifies the instantiation of the information model. The number is composed of an identification of the issuer organization and a unique number assigned by this organization." />
+      <alias value="Identificatienummer" />
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-780" />
+        <comment value="IdentificationNumber" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <short value="SymptomName" />
+      <definition value="Name and code of the symptom via selection from a code list. Represents the symptom which a condition in the patient presents itself." />
+      <alias value="SymptoomNaame" />
+      <binding>
+        <strength value="required" />
+        <description value="Identification of the symptom." />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.26.11.35--20240501114333" />
+      </binding>
+      <mapping>
+        <identity value="cio-dataset-200-beta2-20231214" />
+        <map value="cio-dataelement-20-901" />
+        <comment value="SymptomName" />
+      </mapping>
+    </element>
+    <element id="Observation.code.text">
+      <path value="Observation.code.text" />
+      <short value="FurtherSpecificationSymptomName" />
+      <definition value="More detailed specification of the symptomName that has been selected from the code list, because that list does not (yet) contain the required level of detail." />
```

Comment:
'symptomName' is very ugly; please let that be changed to 'name of the symptom' (compare with FurtherSpecificationDiagnosisName in the HypersensitivityIntolerance building block) in the data set itself, and then update this description accordingly.

Reply:
Raised this with the IA. Will adjust once the data set is adjusted.

Reply:
IA's do not want to change the description as it is the literal description used in the zib. Conclusion: I will not change it.

Reply:
Has this typo already been communicated to the zib centre, so it can be fixed in a later version?